### ISSUE TITLE
Is this intended to support all the file formats that biome currently supports?

### ISSUE THREAD

**Issue by rawrmonstar** [sublimelsp/LSP-biome#63]
Hi! I am trying to get lsp-biome to format and lint css files since biome supports that.

in `LSP-biome.sublime-settings` I've had to update the selector to include "source.css"

e.g., 

```json
{
  "selector": "source.js | source.ts | source.jsx | source.tsx | source.js.jsx | source.js.react | source.ts.react | source.css"
}
```

It shows up, but it doesn't actually report any issues or format the document. Is it intended that this plugin only works for js (and ts) code?

**Comment by rawrmonstar** [comment]
Nevermind, I think it works. My configuration file was named `biome.jsonc` (normally supported by biome), but if I rename it to `biome.json` instead it does report warnings.
